tools/Install/setup.sh

```diff
@@ -25,8 +25,11 @@ set -o nounset  # Exit the script if any uninitialized variable is used.
 
 CLONE_URL=${CLONE_URL:- 'https://github.com/xmos/avs-device-sdk.git'}
 
-PORT_AUDIO_FILE="pa_stable_v190600_20161030.tgz"
-PORT_AUDIO_DOWNLOAD_URL="http://www.portaudio.com/archives/$PORT_AUDIO_FILE"
+# PORT_AUDIO_FILE="pa_stable_v190600_20161030.tgz"
```

Comment:
I you want to leave these old symbols commented out, please include a comment stating the reason so that someone coming across them later will understand why they are still present.